string_functions.js

```diff
@@ -43,12 +42,15 @@ function checkPalindrome(str){
     if(stringToCheck === reverse){
         return "The string "+stringToCheck+" is a palindrome";
     }
-     //if they are not equal, return 'not palindrome'
     return "The string "+stringToCheck+" is not a palindrome";
 
 
 }
 
+
```

Comment:
Can you please add some JS documentation to it!

Reply:
I updated the code, added documentation.

Reply:
okey!Can u please export the lower case function!

Reply:
Exported the function, and updated the code.

Reply:
LGTM..